README.md

```diff
@@ -37,7 +37,7 @@ install.packages("EGRET")
     ## package 'EGRET' successfully unpacked and MD5 sums checked
     ## 
     ## The downloaded binary packages are in
-    ##  C:\Users\lcarr\AppData\Local\Temp\1\RtmpAPk7Z0\downloaded_packages
+    ##  C:\Users\lcarr\AppData\Local\Temp\1\Rtmp8QoZAB\downloaded_packages
```

Comment:
yeah, I'm just saying I like the way it is, and it will change every time someone re-knit's the readme...but, so what? agree?